spyder/plugins/ipythonconsole/utils/kernelspec.py

```diff
@@ -33,6 +33,13 @@
 logger = logging.getLogger(__name__)
 
 
+def validate_interpreter(pyexec):
+    """Check python excutable path and name."""
+    executable_validation = 'python' in osp.basename(pyexec)
```

Comment:
Maybe it is better to use startswith( to ensure it starts with python

Reply:
Why did you mark this as solved @ccordoba12 ?

`executable_validation = 'python' in osp.basename(pyexec)`

Should be

`executable_validation = osp.basename(pyexec).stratswith('python')`

Reply:
Since I rebased I think the initial commit where I did the change was removed. I re-did the change in the last commit